mis_builder_cash_flow/models/mis_cash_flow_forecast_line.py

```diff
@@ -0,0 +1,31 @@
+from odoo import fields, models
+
+
+class MisCashFlowForecastLine(models.Model):
+
+    _name = 'mis.cash_flow.forecast_line'
+    _description = 'MIS Cash Flow Forecast Line'
+
+    date = fields.Date(
+        required=True,
+        index=True,
+    )
+    account_id = fields.Many2one(
+        comodel_name='account.account',
+        string='Account',
+        required=True,
+    )
+    name = fields.Char(
+        required=True,
+        default='/',
+    )
+    balance = fields.Float(
+        required=True,
+    )
+    company_id = fields.Many2one(
```

Comment:
Maybe a constrain between the company and the company on the account could be added

Reply:
Thanks, done